.github/workflows/close_issue_questions.yml

```diff
@@ -0,0 +1,28 @@
+name: Close Issue on Specific Text
+
+on:
+  issues:
+    types: [opened]
+
+permissions:
+    issues: write
+
+jobs:
+  close_issue_if_forbidden_text_found:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close issue with specific text
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const issueBody = context.payload.issue.body;
+            const forbiddenText = 'Please do not submit this issue.';
```

Comment:
I'd be more specific, maybe add a label when creating the issue and check it here? Or at least check for more text, e.g. that it also includes "This issue tracker is reserved for bug reports and feature requests.". Maybe I'm being paranoiac but I'd like to avoid issues closed by mistake.